R/barchart.R

```diff
@@ -301,9 +303,11 @@ Bar <- function(x,
                       width = marker.border.width))
 
         # add invisible line to force all categorical labels to be shown
+        tmp.min <- if (any(is.finite(chart.matrix))) min(chart.matrix, na.rm = TRUE)
+                   else x.bounds.minimum 
```

Comment:
Will this handle `-Inf` in `chart.matrix` sensibly?

Reply:
Thanks for picking that up. I've changed this line (and the corresponding lines in the other charts to
`tmp.min <- if (any(is.finite(chart.matrix))) min(chart.matrix[is.finite(chart.matrix)])`